docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -590,52 +596,50 @@ Confirm Network Functionality
           "link": "http://rest-api:8008/peers"
         }
 
-#. You can also use Sawtooth commands to show peer information.
+#. (Optional) You can also run Sawtooth commands on a validator node to show
+   the other nodes on the network, called `peers`.
 
-   * Use ``sawtooth peer list`` to list the peers of a particular node. For more
-     information, run ``sawtooth peer list --help``.
+   * Use ``sawtooth peer list`` to show the peers of a particular node.
 
    * Use ``sawnet peers list`` to display a complete graph of peers on the
-     network. For more information, run ``sawnet peers list --help``.
+     network.
 
-#. Submit a transaction on the first validator node. This example sets a key
-   named ``MyKey`` to the value 999.
+#. Submit a transaction to the REST API on the first validator node. This
+   example sets a key named ``MyKey`` to the value 999.
 
-   * Docker:
+   * Docker: Use the shell container to run the following command.
 
         .. code-block:: console
 
-        # intkey set --url http://sawtooth-rest-api-default-0:8008 MyKey 999
+           # intkey set --url http://sawtooth-rest-api-default-0:8008 MyKey 999
```

Comment:
Are these spacing changes just for consistency?

Reply:
The RST indenting was wrong.  This change ensures that the command appears inside the code-block box.